src/services/document-service.js

```diff
@@ -23,6 +23,8 @@ import { DOCUMENT_VERIFIED_RELEVANCE_POINTS } from '../domain/constants.js';
 import { documentDBSchema, documentRevisionDBSchema } from '../domain/schemas/document-schemas.js';
 import { checkRevisionOnDocumentCreation, checkRevisionOnDocumentUpdate } from '../utils/revision-utils.js';
 import { createSectionRevision, extractCdnResources, validateSection, validateSections } from './section-helper.js';
+import { isRoomOwner } from '../utils/room-utils.js';
+import permissions, { hasUserPermission } from '../domain/permissions.js';
```

Comment:
Some crazy 💩 happened here 😅

Reply:
:scream: Let's pretend that never happened

Reply:
💊 ... What never happened?

Reply:
:sunglasses: